### PR TITLE
Package dependencies

### PR DESCRIPTION
**Pull request by huntermontell**

### CODE REVIEW

**Review by joshtorres — CHANGES_REQUESTED** [review]
Please fix tests

**Comment by wolfrednicolas** [comment]
@huntermontell looks like https://github.com/spatie/data-transfer-object needs `php 8` for version `3.x`, but the prefer-lowest test is in php `7.4.16`

**Comment by wolfrednicolas** [comment]
@huntermontell 
also the class `FlexibleDataTransferObject` does not exist for the newest version

https://github.com/spatie/data-transfer-object/tree/master/src

but for the v2 it does

https://github.com/spatie/data-transfer-object/tree/v2/src

so here you have a conflict between the versions and the PHP

multiples solutions here

1- upgrading everthing here to php 8, remove the `"php": "^7.4|^8.0"` to `"php": "^8.0"` in the `composer.json`  .... also remove the lowest version for testing. and rewrite the tests already made
2- other approach would be downgrade the `spatie/data-transfer-object` to `2.x`

it's a tough one ..... ask to the team what will be the best approach

**Comment by joshtorres** [comment]
@phuclh do you have any input on this since you worked on it?

File: composer.json [hunk 1]
```diff
@@ -18,15 +18,15 @@
     "require": {
         "php": "^7.4|^8.0",
         "google/apiclient": "^2.7",
-        "schulzefelix/laravel-search-console": "^1.7",
+        "schulzefelix/laravel-search-console": "^1.7.1",
         "skagarwal/google-places-api": "^1.7",
-        "spatie/data-transfer-object": "^2.8",
-        "spatie/valuestore": "^1.2",
+        "spatie/data-transfer-object": "^3.1",
```
Comment (phuclh) [inline composer.json]: We can only use v2 at this time. The v3 only supports PHP8

**Review by phuclh — COMMENTED** [review]
@joshtorres

**Comment by joshtorres** [comment]
Ok let's downgrade to v2 @wolfrednicolas Please verify everything is still working

**Comment by wolfrednicolas** [comment]
Ready to merge @joshtorres

**Review by joshtorres — APPROVED** [review]
Thank you!

**Pull request merged**